src/components/pager/pager.component.tsx

```diff
@@ -225,6 +225,9 @@ export const Pager = ({
           onKeyDown={handleKeyDown}
           data-element="page-select"
           id={pageSizeSelectId}
+          {...(!showPageSizeLabelAfter && !showPageSizeLabelAfter
```

Comment:
question: Is this a mistake that you're checking `!showPageSizeLabelAfter` twice?

Reply:
yes, it is - should be checking ! them both. Worrying that this isn't causing a test failure 🤦